cdap-ui/app/cdap/components/AppDetailedView/Tabs/HistoryTab.js

```diff
@@ -116,7 +116,7 @@ export default class HistoryTab extends Component {
       });
   }
 
-  renderTableBody(history) {
+  renderTableBody = (history) => {
```

Comment:
nit: If we are touching this file can we remove the constructor and have things as class properties (like `ProgramTable/index.js`)